src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -11,6 +11,7 @@ import {
   TemplateResult,
 } from "lit-element";
 import "../../../components/ha-circular-progress";
+import { showMediaBrowserDialog } from "../../../components/media-player/show-media-browser-dialog";
```

Comment:
And we can open the media browser now? 😜

Reply:
I have no idea how that got in there. Must have accidentally hit tab at the wrong moment in vscode or something...

Reply:
Why didn't the commit hook linting warn me about this, btw? Is it not the same as the github action?

Reply:
We don't do commit hook linting (yet), we just run prettier